tests/unit/set.c

```diff
@@ -69,6 +69,36 @@ int main(void) {
     if (igraph_set_contains(&set, 42) || !igraph_set_contains(&set, 7)) {
         return 3;
     }
+    
+    /* igraph_set_remove */
+    // remove element in the middle
+    igraph_set_remove(&set, 10);
+    if (igraph_set_size(&set) != 20 || igraph_set_contains(&set, 10)) {
+        return 5;
```

Comment:
For future tests, it's fine to use `IGRAPH_ASSERT`. Returning a non-zero exit code is an old style that we're abandoning. This test will eventually get rewritten to use `IGRAPH_ASSERT`.